bin/test_registrar

```diff
@@ -18,7 +18,7 @@ clean_devices=$(fgrep 'Z"' ${TEST_SITE}/registration_summary.json | wc -l)
 exit_status=0
 
 echo Found ${clean_devices} clean devices.
-[ "${clean_devices}" == 4 ] || exit_status=1
```

Comment:
Can we make this automatic?

Reply:
What do you mean by automatic?

Reply:
Automatic = Can we derive the 4 from a calculation based on the test inputs

Reply:
I'm semi ok with this if we make it less of a hunt to figure out why it's 3, instead of it just being a bare int by itself in a file that calculates the rest.  For example

# Clean devices are: SNS-4 GAT-123 AHU-22
[ "${clean_devices}" == 3 ].....

Reply:
I'm really not sure it's worth it in this case. The logic to calculate that
for just this one case would be relatively complicated (find all the files
that don't match a very specific pattern that just matches one file), and
then likely would end up bring more fragile than just having the raw
expected value -- and then you'd also have to worry about an inadvertent
change in the input data changing the expected and actual value for the
test...

Reply:
Ok -- added named constant with detail about why it is what it is!